### PR TITLE
[LC-892] Update third-party libraries

### PR DESCRIPTION
**Pull request by metalg0su**
(coupled: https://github.com/icon-project/loopchain/pull/479)

# Goal
- Update third-party libraries to the latest one (2020-03-02)

## Target Libraries
|Package|Version|Latest|
|-----------|-----------|--------|
|grpcio|1.25.0|1.27.2|
|grpcio-tools|1.25.0|1.27.2|
|jsonschema|2.6.0|3.2.0|
|protobuf|3.7.0|3.11.3|
|requests|2.20.0|2.23.0 (REMOVED)|
|sanic|18.12.0|19.12.2|
|Sanic-Cors|0.9.6|0.10.0|

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
 gunicorn>=20.0,<=20.0.4
-grpcio==1.25.0
-grpcio-tools==1.25.0
-protobuf==3.7.0
-aiohttp==3.4.4
+grpcio==1.27.2
+grpcio-tools==1.27.2
+protobuf==3.11.3
+aiohttp==3.6.2
 secp256k1==0.13.2
```
Comment (yakkle) [inline requirements.txt]: why this library exists anywhere? 😱

Comment (yakkle) [reply requirements.txt]: I think `secp256k1` no more needed.

**Comment by yakkle** [comment]
update like loopchain requirements.
```
jsonrpcclient[aiohttp]==3.3.5
```

**Review by windies21 — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (metalg0su) [reply requirements.txt]: Checked no usage.
Thank you!

**Comment by metalg0su** [comment]
> update like loopchain requirements.
> 
> ```
> jsonrpcclient[aiohttp]==3.3.5
> ```

Thank you. 
I removed `aiohttp` from `requirements.txt`. Now It is installed via `jsonrpcclient`.
Please check.

**Review by yakkle — APPROVED** [review]

**Review by windies21 — APPROVED** [review]

**Pull request merged**